### PR TITLE
Add "test_ovs_new_flows_added_after_restart"

### PR DESCRIPTION
**Pull request by tkuterina**
Testcase "test_ovs_new_flows_added_after_restart" and function "get_current_cookie" were added in mos_tests\neutron\python_tests\test_ovs_restart.py file

### CODE REVIEW

File: mos_tests/nova/nova_tests.py [hunk 1]
```diff
@@ -466,3 +484,106 @@ def test_2238776_NetworkConnectivityToVMDuringLiveMigration(self):
...
+            sleep(1)
+            inst = self.nova.servers.get(inst.id)
+        self.assertEqual(inst.status, 'ACTIVE')
+        ping.wait()
+        output = ping.stdout.read().split('\n')[-3].split()
+        packets = {'transmitted': int(output[0]), 'received': int(output[3])}
+        loss = packets['transmitted'] - packets['received']
+        if loss > 5:
+            raise AssertionError(
+                "Packets loss exceeds the limit, {} packets were lost".format(
```
Comment (TimurNurlygayanov) [inline mos_tests/nova/nova_tests.py]: msg = "Packets loss exceeds the limit, {} packets were lost"
raise AssertionError(msg.format(loss))

File: mos_tests/nova/nova_tests.py [hunk 2]
```diff
@@ -466,3 +484,106 @@ def test_2238776_NetworkConnectivityToVMDuringLiveMigration(self):
...
+        self.nova.servers.live_migrate(inst, new_hyper, block_migration=False,
+                                       disk_over_commit=False)
+        inst = self.nova.servers.get(inst.id)
+        timeout = 10
+        end_time = time() + 60 * timeout
+        while getattr(inst, "OS-EXT-SRV-ATTR:hypervisor_hostname") != \
+                new_hyper:
+            if time() > end_time:
+                raise AssertionError(
+                    "Hypervisor is not changed after live migration")
```
Comment (TimurNurlygayanov) [inline mos_tests/nova/nova_tests.py:562]: msg = "Hypervisor is not changed after live migration"
raise AssertionError(msg)
